stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -117,35 +117,6 @@ bimg  = randn(n,2)/2
         end
     end # for eltyb
 
-@testset "Test diagm for vectors" begin
```

Comment:
I moved the following three test suites out of the loop over `eltya`, since they are independent of it. The indentation seems to indicate that this was already realized, but the placement of the tests nevertheless includes them in the loop over `eltya`.